Northstar.Client/mod/resource/ui/menus/panels/sb_map_grid.res

```diff
@@ -0,0 +1,795 @@
+"resource/ui/menus/panels/sb_map_grid.res"
```

Comment:
Is this a new vanilla file that's been modified or completely written from scratch?

Reply:
not vanilla.

Reply:
Aight, keeping this open until vanilla file is added by a maintainer (like me).


If there's no activity within a reasonable time frame, feel free to poke me so that the PR isn't overlooked ^^

Reply:
`map_grid_button.res` is not a vanilla file, it is a file added by northstar. Nothing needs to be committed as respawn

Reply:
> Wait what?

`map_grid_button.res` (alongside `map_grid.res`) was added in https://github.com/R2Northstar/NorthstarMods/pull/194 and is not a vanilla file. It does not exist in vpks. @AlphaGaming7780 may be thinking that files included in base northstar are vanilla files, which is not the case.